pkg/sql/logictest/testdata/logic_test/partial_index

```diff
@@ -605,18 +605,90 @@ INSERT INTO k VALUES (1, 1), (6, 6)
 statement ok
 TRUNCATE k
 
-query II
+query II rowsort
 SELECT * FROM k@a_b_gt_5 WHERE b > 5
 ----
 
 statement ok
 INSERT INTO k VALUES (1, 1), (7, 7)
 
-query II
+query II rowsort
 SELECT * FROM k@a_b_gt_5 WHERE b > 5
 ----
 7  7
 
+# Test partial indexes with an ENUM in the predicate.
```

Comment:
add a subtest

Reply:
I added it, but what is the benefit? `pkg/sql/logictest/logic.go` doesn't explain much. Is it just the additional label on failures like: `---FAIL: TestLogic/local/partial_index/enum`. I can't seem to run only the tests in the subtest with `make test PKG='./pkg/sql/logictest' TESTS='TestLogic/local/partial_index/enum'`—it runs all the tests in the `partial_index` file.

Reply:
yeah the additional label, and you can run subtests with the `SUBTESTS=...` flag to the make testlogic incantation